modules/ROOT/partials/misc/ui-languages.adoc

```diff
@@ -1,8 +1,11 @@
+[[supportedlanguages]]
 === Supported Languages
 
-{% if forModuleLoaders == true %}
+ifeval::[{forModuleLoaders} == true]
+[cols='76,7,20,0']
```

Comment:
The markup below in this table is incorrect which is actually causing 4 columns to render. Basically it has an extra `|` at the end of each row that shouldn't be there. Additionally, this cols spec should only have 3 cols and the widths are too squashed for the last 2 cols which leads to a somewhat poor experience.

- [ ] Fix the conversion issues in this file and adjust the column widths so the last 2 cols aren't so squashed.

Reply:
ok, I copied this: `[cols="<2,^2,^2",options="header"]` from the TinyMCE 6 branch, I only changed the center column to have the last 2 cols less squashed, I'll finish doing the other fixes and I'll commit it

Reply:
done